internal/painter/gl/painter.go

```diff
@@ -128,7 +136,7 @@ func (p *painter) createProgram(shaderFilename string) Program {
 	// Why a switch over a filename?
 	// Because this allows for a minimal change, once we reach Go 1.16 and use go:embed instead of
 	// fyne bundle.
-	sources := shaderSources[shaderFilename]
+	sources := shaderSourceNamed(shaderFilename)
 	vertexSrc, fragmentSrc := sources[0], sources[1]
```

Comment:
If the only use case is to split this in vertex and fragment shader, why not return two []byte{} instead of one [2][]byte{}.

Reply:
Good call. I'll get that sorted tomorrow.